### DISCUSSION TITLE
Any Snowpack integration examples?

### DISCUSSION THREAD

**Discussion by amankkg** [lingui/js-lingui#924] [Ideas]
I'm researching CRA alternatives and Snowpack looks very appealing. So far I've reproduced most of our app stack using snowpack (MUI, TS, new JSX transform, etc.). Now I need to check if it's possible to configure dev setup for Lingui.

AFAIU, there is no clear way to replace webpack loader in order to improve DX. Other parts (dynamic imports and macro) are good to go (almost) out of the box: one has to add snowpack's babel plugin with macros babel plugin.

I wonder if anyone tried to do the same?

**Comment by amankkg** [comment]
I guess we may need a snowpack plugin to fulfill the functionality provided by lingui webpack loader. I wonder if it's the right direction...

**Comment by amankkg** [comment]
OK, here is my early results https://github.com/amankkg/frontend-template

**Comment by amankkg** [comment]
Got a draft version of snowpack plugin working here https://github.com/amankkg/frontend-template/commit/1b99583fff49735aaa82cf3a58254bac020a7387

So far it only supports default config location and only a `.po` format

**Reply by semoal** [reply]
I could help on the plugin and probably if you are agree we could include it as official plugin inside lingui monorepo.

**Reply by amankkg** [reply]
Sure, I don't mind contributing it to lingui. It's actually a copy-paste of the official webpack loader 😃

**Reply by semoal** [reply]
Aman, did you get it working? Do you want to create a pull request and comment there the pending tasks? I could help :)

**Reply by amankkg** [reply]
Yeah it works, I'll create a PR with new entry `src/packages/snowpack-plugin-lingui`

Maybe we can continue this discussion in issue?

**Reply by amankkg** [reply]
So original discussion remains general about tips on snowpack integration

**Reply by amankkg** [reply]
Here it is https://github.com/lingui/js-lingui/issues/944
